milestone2.ts

```diff
@@ -83,34 +83,39 @@ const userData: Volunteer[] = [
   Hint: user data is stored in the userData object above. 
 */
 type GetNumber = (data: Volunteer[]) => number;
-let findAverage: GetNumber; // Define lambda function here
+let findAverage: GetNumber = (data) => {
+  const ages: number[] = data.map(x => x.age);
+  let sum: number = ages.reduce((prev, cur) => prev + cur);
+  return sum/data.length;
```

Comment:
creative way to find the average, you could also skip the mapping and just use a reduce function where you add the age field from each of them.